### PR TITLE
Test.txt

### PR DESCRIPTION
**Pull request by KathanPathak06**
First file for testing

### CODE REVIEW

File: Test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Kem chho?
```
Comment (KaranJoshi03) [inline Test.txt:1]: Ekdum Majama

**Review by zeel1998 — APPROVED** [review]
Nathi majama

**Review by KaranJoshi03 — APPROVED** [review]

File: Test.txt (hunk 1 above)
Comment (JayVyas01) [reply Test.txt:1]: **Jarai maja nthi
**

**Review by JayVyas01 — CHANGES_REQUESTED** [review]
file na naam mathi space kadh.

**Review by JayVyas01 — APPROVED** [review]

File: Farithi.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Have to barobar ne?
```
Comment (JayVyas01) [inline Farithi.txt:1]: Ha done

Comment (zeel1998) [inline Farithi.txt:1]: Na na 
Tane aavdta hoy eva 3 computer scientist na name lakh

Comment (zeel1998) [reply Farithi.txt:1]: Na na
Tane aavdta hoy eva 3 computer scientist na name lakh

**Review by zeel1998 — CHANGES_REQUESTED** [review]
Na na
Tane aavdta hoy eva 3 computer scientist na name lakh

File: Farithi.txt (hunk 2 above)
Comment (KathanPathak06) [reply Farithi.txt:1]: Mari same j 3 jana che

**Review by JayVyas01 — APPROVED** [review]

**Review by zeel1998 — APPROVED** [review]

**Pull request merged**